### PR TITLE
feat: pass decider manager

### PR DESCRIPTION
**Pull request by syuhei176**

### CODE REVIEW

File: src/initialize.js [hunk 1]
```diff
@@ -3,11 +3,14 @@ import LightClient, {
   types,
   ethWallet,
   db,
+  ovm,
   ethContract,
   StateManager,
   SyncManager,
   CheckpointManager
 } from 'wakkanay-plasma-light-client'
+import config from '../config.local'
```
Comment (syuhei176) [inline src/initialize.js]: how do we change config file by env?

Comment (tkmct) [reply src/initialize.js]: what about create a helper function named `loadConfig` and do something like

```
import dotenv from 'dotenv'
import config from '../config.local'
dotenv.configure()

function loadConfig() {
  const config = config;
  
  // override config using process.env loaded by dotenv.configure()

  return config
}

```

Comment (syuhei176) [reply src/initialize.js]: I want to prepare `config.local`, `config.kovan` and `config.main`. And load it dynamically.

Comment (syuhei176) [reply src/initialize.js]: updated

**Pull request merged**